semgrep/semgrep/config_resolver.py

```diff
@@ -439,6 +438,16 @@ def parse_config_string(
             f"Empty configuration file {filename}",
             code=UNPARSEABLE_YAML_EXIT_CODE,
         )
+    try:
+        rule_definitions = [
+            rule
+            for rule_model in json.loads(contents)
+            for rule in rule_model["definition"]["rules"]
+        ]
+        # we pretend it came from YAML so we can keep later code simple
```

Comment:
Despite this, parsing local YAML rules is now broken somehow. Will have to come back here and fix that.